workshops/rock_paper_scissors/README.md

```diff
@@ -0,0 +1,294 @@
+```py
+player = False
+```
+
+Now, let's write some code so the player can enter their name.
+```py
+name = input("Enter your name:")
+```
+
+Also to make a scoring system, we'll write the following code:
+```py
+You = 0
+PC = 0 
+```
+
+Next, we want to give an option to the player to reset the game.
+```py
+print("Type 'reset' to reset score")
+```
+
+Here's the code so far:
+```py
+from random import randint
+
+t = ["r", "p", "s"]
+
+computer = t[randint(0,2)]
+
+player = False
+
+name = input("Enter your name:")
+print("Type 'reset' to reset score")
+
+You = 0
+PC = 0
+```
+
+### 3)Keeping the game in a loop
+![loop](https://cloud-nmra250be.vercel.app/loop.gif)
+
+For this, I'll use a `while` loop but you can also use `for` loops!
+
+So we'll just type:
+```py
+while Player == False: 
+```
+Now we let the player choose either `Rock` `Paper` or `Scissors`
+```
+player = input("Rock, Paper, Scissors?(r,p,s)")
+```
+
+There are 5 possibilities:
+1. The player chooses `Rock`
+2. The player chooses `Paper`
+3. The player chooses `Scissors`
+4. The word entered is invalid
+5. The player enters `reset`
+
+### 4) Producing separate outcomes for all possibilities
+
+To make things easier I will be making functions to print if the player wins or loses.
+First I will write `message = ""` This will help us by printing different message by calling the same function.
+
+Now I will be making a function that will be called only when the player wins. To do that enter the following code:
+```py
+def win():
+  global You
+  You+=1
+  print (message)
+  print('Computer =', PC, '\n', name, '=', You)
+```
+Now we will make a function that will be called only when the player will lose.
+```py
+def lose():
+  global PC
+  PC+=1
+  print (message)
+  print('Computer =', PC, '\n', name, '=', You)
+```
+Now we have to call these functions at the right places.
+
+To do that enter the following code:
+```py
+while player == False:
+    player = input("Rock, Paper, Scissors?(r,p,s)")
+    if player == computer:
+        print("Tie!")
+        print('Computer =',PC)
+        print(name,'=',You)
+    elif player == "r":
+        if computer == "p":
+            message = "You lose!","Paper covers Rock"
+            lose()
+        else:
+            message ="You win!", "Scissors cuts Paper"
+            win()
+    elif player == "p":
+        if computer == "s":
+            message = "You lose!","Paper covers Rock"
+            lose()
+        else:
+            message ="You win!", "Scissors cuts Paper"
+            win()
+    elif player == "s":
+        if computer == "r":
+            message = "You lose!","Paper covers Rock"
+            lose()
+        else:
+            message ="You win!", "Scissors cuts Paper"
+            win()
+    elif player == "reset":
+      You=1*0
+      PC=1*0
+      print("The score has been reset!")
+    else:
+        print("That's not a valid play. Please select a valid option!")
+```
+#### Explanation:
```

Comment:
Going with my previous suggestion, if  you do that then this whole explanation section can probably be removed